njbrunner/covid-19
# Add logout button

**Note:** Add login and signup redirect pull request should be reviewed and merged before this pull request.

**Changes:**
Added logout button (conditionally shows based on user login status.

**Testing:**
- login user
- see new icon in top right of nav bar
- hover over icon and select logout
- app should logout user and redirect to login page
